hack/add-plugin.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+COREDNS_PATH=$1
+PLUGIN_PATH=$(readlink -f "$(dirname "$0")/..")
+
+# Add/update the plugin details in plugin.cfg file.
+if grep -wq "ocp_dnsnameresolver" $COREDNS_PATH/plugin.cfg
+then
+    sed -i "/ocp_dnsnameresolver/c\ocp_dnsnameresolver:github.com/openshift/coredns-ocp-dnsnameresolver" $COREDNS_PATH/plugin.cfg
+else
+    sed -i "/cache:cache/a ocp_dnsnameresolver:github.com/openshift/coredns-ocp-dnsnameresolver" $COREDNS_PATH/plugin.cfg
+fi
```

Comment:
I kept the update portion as a backup so that even if the plugin was erroneously added with a wrong package previously, it'll update it to the correct one.